tests/webview/ui/test_content.py

```diff
@@ -163,6 +163,22 @@ def test_get_this_book(webview_base_url, selenium):
     assert offline_zip_displayed == downloads.is_offline_zip_available
 
 
+@markers.webview
+@markers.nondestructive
+def test_section_title(webview_base_url, selenium):
```

Comment:
Be more specific with the test name. `test_section_title_for_no_markup`

Reply:
Changed the test name to `test_section_title_for_no_markup`